remix/app/routes/_app._index.tsx

```diff
@@ -1,12 +1,19 @@
 import { User } from '@prisma/client';
 import { useOutletContext } from '@remix-run/react';
 
+import AppLink from '../components/AppLink';
+
 export default function Index() {
   const [user] = useOutletContext() as [User];
 
   return (
     <>
```

Comment:
I'm just curious but do we need the fragment here? Cause I thought fragments are used to just group childs that not have a parent, but we have a `section` parent here 🤔

Reply:
I means the

```typescript
    <>
      <section>
      </section>
    </>
```

Reply:
> LGTM! I like the way you handle `toast` 😄

Thanks for that, you can implement the way I do it here as `Remix` and `NextJs` are both `React` base  framework!

Reply:
For the fragment there, actually this come from my own behavior, as the old version React is define the file as `.js`, so I just find the way to tell which `.js` file is `React` component, by using the `React Fragment` as the mark for me.

Nowadays, we don't actually need that as we define file as `.jsx` or `tsx`

I will remove it, thanks for asking me that, it help me to improve too 😅

Reply:
Improved here: [07415ce](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/07415cea75c043a849c912f59b3c4428548ba754)

Reply:
Ohh #til 👀